### PR TITLE
params: Add Constantinople block to AllCliqueProtocolChanges

### PR DESCRIPTION
**Pull request by enriquefynn**

### CODE REVIEW

**Comment by enriquefynn** [comment]
small change for #18162

**Comment by enriquefynn** [comment]
ref. #18154

File: params/config.go [hunk 1]
```diff
@@ -118,7 +118,7 @@ var (
 	//
 	// This configuration is intentionally not using keyed fields to force anyone
 	// adding flags to the config to also have to set these fields.
-	AllCliqueProtocolChanges = &ChainConfig{big.NewInt(1337), big.NewInt(0), nil, false, big.NewInt(0), common.Hash{}, big.NewInt(0), big.NewInt(0), big.NewInt(0), nil, nil, nil, &CliqueConfig{Period: 0, Epoch: 30000}}
+	AllCliqueProtocolChanges = &ChainConfig{big.NewInt(1337), big.NewInt(0), nil, false, big.NewInt(0), common.Hash{}, big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), nil, nil, &CliqueConfig{Period: 0, Epoch: 30000}}
```
Comment (karalabe) [inline params/config.go]: Please add it to the `AllEthashProtocolChanges` too.

**Comment by holiman** [comment]
`git bisect` identified this commit as breaking some testcases
```
[user@work go-ethereum]$ go test ./core/ -run TestLastBlock
--- FAIL: TestLastBlock (0.01s)
    blockchain_test.go:209: Failed to insert block: invalid merkle root (remote: f89cc0e6aef84e24cb27012b28042d7aa3a54a38b8aa5d671382a2d5ea17fa2d local: e7cb9d2fd449f7bd11126bff55266e7b74936f2f230e21d44d75c04b7780dfeb)
FAIL
FAIL	github.com/ethereum/go-ethereum/core	0.182s
[user@work go-ethereum]$ git bisect bad
9d5e3e06370dace249305d91624cb83cabf3c25f is the first bad commit
commit 9d5e3e06370dace249305d91624cb83cabf3c25f
Author: Enrique Fynn <me@enriquefynn.com>
Date:   Thu Nov 22 14:03:50 2018 +0100

    params: add Constantinople block to AllXYZProtocolChanges (#18162)
    
    * params: Add Constantinople block to AllCliqueProtocolChanges
    
    * params: Add Constantinople block to AllEthashProtocolChanges

:040000 040000 644fd45b69e5606621960ab6d1c6ab8324ef9d0f 41089a93844c9ad2726fa91e0d93e6ccb369d06c M	params

```

**Pull request merged**